extensions/ql-vscode/src/remote-queries/remote-query-result.ts

```diff
@@ -3,6 +3,7 @@ import { DownloadLink } from './download-link';
 export interface RemoteQueryResult {
   executionEndTime: Date;
   analysisSummaries: AnalysisSummary[];
+  artifactStorageDir: string;
```

Comment:
Nit, but I think this should be called `artifactsStorageDir` (plural for artifacts)

Or, if it's made part of the `RemoteQuery` (rather than `RemoteQueryResult`) as a I mention in another comment perhaps it could just be called `storageDir` since it's meant to contain general query data.

Reply:
Removed.